Rocket.Chat/Controllers/Chat/ChatDataController.swift

```diff
@@ -206,9 +206,9 @@ final class ChatDataController {
     func update(_ message: Message) -> Int {
         for (idx, obj) in data.enumerated()
             where obj.message?.identifier == message.identifier {
-                if obj.message?.updatedAt?.timeIntervalSince1970 == message.updatedAt?.timeIntervalSince1970 {
-                    return -1
-                }
+                //if obj.message?.updatedAt?.timeIntervalSince1970 == message.updatedAt?.timeIntervalSince1970 {
+                //   return -1
+                //}
```

Comment:
Wondering why did you comment this line... this will cause tons of updates in cells that doesn't need to be updated.

Reply:
This is what was causing the issue where edited things would not be refreshed on the app and, afterwards, receiving a message would cause a crash. Please check my post from 22 days ago (near commit 4ae3eb6)

Reply:
Yeah, I can see that... but we can't remove this verification. You need to update the `updatedAt` value of your message in order to update the cell. Makes sense?

Reply:
It does, but I already do that in [here](https://github.com/RocketChat/Rocket.Chat.iOS/pull/1048/files#diff-914164611cb69f4158d88d4fb8927685R409).

Maybe this should be done in other place?

Reply:
Well... have you tried uncommenting this code and trying to edit a message?